src/release-notes/sssd-2.10.0.rst

```diff
@@ -0,0 +1,126 @@
+  under ``root`` (if needed for some reason): besides changing ``User/Group=``
+  options, some other tweaks of systemd service files are required.
+
+  A legacy method to configure a service user - sssd.conf ``user`` option - is
+  now deprecated and its support isn’t built by default. It can be enabled using
+  ``--with-conf-service-user-support`` ``./configure`` option if needed (for
+  example, due to backward compatibility requirements of stable releases).
+
+  Further, no matter if SSSD is built ``--with-sssd-user=sssd`` or
+  ``--with-sssd-user=root``, when it's configured to run under ``root`` (in both
+  cases) it still runs without capabilities, the same way as when it's
+  configured to run under ``sssd`` user. The only difference is from the DAC
+  perspective.
+
+  Important note: owner of ``/etc/sssd/sssd.conf`` file (and snippets) should
+  match the user configured to start SSSD service. Upstream spec file changes
+  ownership of existing ``sssd.conf`` to ``sssd`` during package installation
+  for seamless upgrades.
+
+  Additionally, this release fixes a large number of issues with "socket
+  activation of responders" feature, making it operable out-of-the-box when the
+  package is built ``--with-sssd-user=sssd``. Please take a note, that user
+  configured to run main sssd.service and socket activated responders (if used)
+  should match (i.e. if sssd.service is re-configured from upstream defaults to
+  ``root`` then responders services also should be re-configured).
+
+  Downstream package maintainers are advised to carefully inspect changes in
+  ``contrib/sssd.spec.in``, ``src/sysv/systemd/*`` and ``./configure`` options
+  that this release brings!
+
+* sssctl ``cache-upgrade`` command was removed. SSSD performs automatic upgrades
+  at startup when needed.
+
+* Support of ``enumeration`` feature (i.e. ability to list all users/groups
+  using ``getent passwd/group`` without argument) for AD/IPA providers is
+  deprecated and might be removed in further releases. Those who are interested
+  to keep using it awhile should configure its build explicitly using
+  ``--with-extended-enumeration-support`` ./configure option.
+
+* A number of minor glitches of ``sssd-2.10.0-beta1`` around building and
+  packaging were fixed.
+
+* Support of ``--without-infopipe`` ``./configure`` option was dropped. Feature
+  is long time out of experimental state. Since building it doesn't require any
+  additional dependencies, there is not much sense to keep option available.
+  Those who not interested in feature can skip installing sssd-ifp sub-package.
+
+New features
+~~~~~~~~~~~~
+
+* The new tool ``sss_ssh_knownhosts`` can be used with ssh's KnownHostsCommand
+  configuration option to retrieve the host's public keys from a remote server
+  (FreeIPA, LDAP, etc.). This new tool, which is more reliable, replaces
+  ``sss_ssh_knownhostsproxy``. Please consider switching to using the new tool
+  as the old one will be removed.
```

Comment:
@aplopez, building of old `sss_ssh_knownhostsproxy` is disabled by default upstream, right?
We should mention it here.

Reply:
This is a text geared more toward maintainers of SSSD in other distributions rather than end users.

Reply:
I would propose:

The new tool ``sss_ssh_knownhosts`` can be used with ssh's KnownHostsCommand configuration option to retrieve the host's public keys from a remote server (FreeIPA, LDAP, etc.). This new tool, which is more reliable, replaces ``sss_ssh_knownhostsproxy``. The latter is no longer built by default, but its build can be forced with ``configure --with-ssh-known-hosts-proxy``.